ci-operator/step-registry/openshift/upgrade/aws/latestinstaller/openshift-upgrade-aws-latestinstaller-workflow.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "openshift/upgrade/aws/latestinstaller/openshift-upgrade-aws-latestinstaller-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"smarterclayton",
+			"wking",
+			"stevekuznetsov",
```

Comment:
I think we need to [drop some of these][1].

/hold

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/17987/pull-ci-openshift-release-master-step-registry-metadata/1389782447410909184#1:build-log.txt%3A17